pyproject.toml

```diff
@@ -1,3 +1,53 @@
+[build-system]
+requires = ["setuptools>=61", "cffi>=1.3.0", "pkgconfig", "libsecp256k1"]
```

Comment:
```suggestion
requires = ["setuptools>=61", "cffi>=1.3.0"]
```

Reply:
Right, that's a left over from not knowing what I was doing ...